core-bundle/src/Doctrine/Backup/BackupManager.php

```diff
@@ -27,22 +27,32 @@ class BackupManager
 
     private Connection $connection;
     private DumperInterface $dumper;
-    private string $backupDir;
+    private VirtualFilesystemInterface $backupsStorage;
     private array $tablesToIgnore;
     private RetentionPolicyInterface $retentionPolicy;
 
-    public function __construct(Connection $connection, DumperInterface $dumper, string $backupDir, array $tablesToIgnore, RetentionPolicyInterface $retentionPolicy)
+    public function __construct(Connection $connection, DumperInterface $dumper, VirtualFilesystemInterface $backupsStorage, array $tablesToIgnore, RetentionPolicyInterface $retentionPolicy)
     {
         $this->connection = $connection;
         $this->dumper = $dumper;
-        $this->backupDir = $backupDir;
+        $this->backupsStorage = $backupsStorage;
         $this->tablesToIgnore = $tablesToIgnore;
         $this->retentionPolicy = $retentionPolicy;
     }
 
+    public function createNewBackup(\DateTime $dateTime = null): Backup
+    {
+        $now = $dateTime ?? new \DateTime('now');
+        $now->setTimezone(new \DateTimeZone('UTC'));
```

Comment:
What exactly does this imply? Shouldn't we use the configured timezone instead?

Reply:
The timezone is not part of the filename so it has to be UTC.

Reply:
Yes, imho we should use the configured timezone, just as the operating system would do.

Reply:
Storing as UTC makes sense IMO. But we could display it in the local timezone in the list command I think.

Reply:
The filename would still say `20220204121042` instead of `…131042` then.

Reply:
Filenames **MUST** be represented in a consistent way. The only valid choice is UTC.

Reply:
Not sure about that, either. `mysqldump mydb > backup__$(date +%Y%m%d%H%M%S).sql` will also show the date according to the server time zone and not UTC (unless UTC is the server time zone of course).

But if you think that UTC is so important, let‘s just add the timezone offset: `date +%Y%m%d%H%M%S%z`

Then the file would be called `backup__ 20220204131042+0100.sql`. Now the timestamp is showing the expected date and time to the user while also representing the timestamp in a consistent way. (I still think you are overthinking this, though. I‘ve been a server admin for over 20 years and we never had a problem with timestamps in filenames not being UTC. 🤷‍♂️)

Reply:
I just don't think it makes sense to consider the timezone in a file. Why not just store it in UTC like we do ever since the first PR? I don't know why this is suddenly a discussion. It's not related to this PR at all.

Reply:
I have only noticed it now, otherwise I would have wanted to discuss this in the first PR as well. You cannot seriously want people to run `contao:backup:create` and then have `contao:backup:list` tell them that the last backup is one hour old? What kind of UX is this supposed to be?

Reply:
We can adjust the list command of course. But adjusting the file name doesn't fix anything while actually making it worse because when you switch the timezone settings you have no way of finding out when exactly the backup was created.

Reply:
There are 2 options imho:

1. Leave the date untouched and adjust the table header from `Created` to `Created (UTC)`
2. Adjust the date to your system timezone and adjust the table header from `Created` to `Created (foobar)`

Which one do you prefer? :)

Reply:
2) is going to be PITA for the unit tests.

Reply:
Neither is a good solution. We might leave the UTC timestamp in the filename but we surely have to adjust the `Created` column to show the date and time according to the server timezone.

Reply:
Why neither? Both display the timezone, that's what you need?

Reply:
I prefer the second option :)